metroplex-slicerextensions_stable_nightly.cmake

```diff
@@ -16,7 +16,7 @@ if(APPLE)
   dashboard_set(CMAKE_OSX_DEPLOYMENT_TARGET "10.13")
 endif()
 dashboard_set(CTEST_CMAKE_GENERATOR "Unix Makefiles")
-dashboard_set(COMPILER              "g++-7.3.1")      # Used only to set the build name
+dashboard_set(COMPILER              "g++-14.2.1")      # Used only to set the build name
```

Comment:
Stable are still build using the "legacy" environment.

https://github.com/Slicer/DashboardScripts/blob/96a4af8aad65ec298568d53be789c8fe2c647d46/metroplex.sh#L71-L74